cluster-up/cluster/kind-k8s-1.17.0-ipv6/README.md

```diff
@@ -6,6 +6,21 @@ The KubeVirt containers are built on the local machine and are then pushed to a
 
 cluster is brought up with ipv6 support but without flannel or multi nic support
 
+## Prerequisits
+1. kubectl >= 1.16
+1. docker network with ipv6.  
+    To get that you'll have to add the following section to /etc/docker/daemon.json:  
+    ```
+    {
+      "ipv6": true,
+      "fixed-cidr-v6": "2001:db8:1::/64"
+    }
+    ```  
+    and to fully restart docker (systemctl restart docker)  
+    if needed, docker can be tested with:
+    `docker run --rm busybox ip a`  
+    and make sure you get an ipv6 address  
+
```

Comment:
I think it is worth mentioning that, for any reason, if someone managed to install docker on RHEL8 and tries to adjust daemon.json as mentioned 
it wont work due to compatibility issues of docker with RHEL8.

Reply:
@ormergi please spell out the required docker version.

Reply:
Any version of docker on centos/fedora should work.

In case of RHEL8, there is no official support for docker-ce, the workaround to have docker on rhel8 is to install docker-ce el7 package, but with this setup ipv6 wont work.